Sources/SwiftDocC/SwiftDocC.docc/Resources/RenderNode.spec.json

```diff
@@ -2814,11 +2814,13 @@
                 "type": "object",
                 "properties": {
                     "baseType": {
-                        "type": "string"
+                        "type": "string",
+                        "nullable": true
```

Comment:
What's the difference between a property that's nullable and a property that's not required? Is the difference that the property allows for an explicit "null" value in addition to interpreting a missing value as a null value?

I only found 12 properties marked as nullable in the various OpenAPI specs but I feel like we have more optional values than that. Are there more places that we should update?

Reply:
Yes, it looks like most of the objects should be marked `nullable: true` explicitly. I would prefer it this would be addressed in a follow-up PR.